src/Operations.ts

```diff
@@ -894,7 +894,7 @@ export function normalizeAddedIndexes(indexes: ColumnRowIndex[]): ColumnRowIndex
     return indexes
   }
 
-  const sorted = indexes.sort(([a], [b]) => (a < b) ? -1 : (a > b) ? 1 : 0)
+  const sorted = [...indexes].sort(([a], [b]) => a-b)
```

Comment:
this code was leaking change to the arguments coming from API (sorting function arguments)